frontend/public/actions/k8s.ts

```diff
@@ -60,22 +60,31 @@ export const startAPIDiscovery = () => (dispatch) => {
       if (res.status.allowed) {
         // eslint-disable-next-line no-console
         console.log('API discovery method: Watching');
+        // Always dispatch an initial call
+        dispatch(getResources());
         // Watch CRDs and dispatch refreshAPI action whenever an event is received
         dispatch(
           watchK8sList(
             reduxID,
             {},
             CustomResourceDefinitionModel,
-            // Only re-run API discovery on added or removed CRDs.
-            (_id: string, events: K8sEvent[]) =>
-              events.some((e) => e.type !== 'MODIFIED') ? getResources() : _.noop,
```

Comment:
This code works only because this callback was strang or at least unexpected called with different values...

First with the CRD list, and then with update-events. This might be a bug in `watchK8sList`... But I didn't want change this here.

`!== 'MODIFIED'` calls getResources also for CRDs, because non of them has any `type` in the resource!

That's the reason I added a `getResource` above and changed the type check in the callback to `type === 'ADDED' || type === 'DELETED` below!